docs/aerodromes/classd/Moorabbin.md

```diff
@@ -21,49 +24,81 @@ MB ADC is responsible for the Class D airspace in the MB CTR `SFC` to `A025`.
 
 Refer to [Class D Tower Separation Standards](../../../separation-standards/classd) for more information.
 
+---
+
 ## Runway Modes
-Equal preference is given to the following runway modes:
 
-- Runways 35L & 35R
-- Runways 17L & 17R
-- Runways 31L & 31R
-- Runways 13L & 13R
+Preferred runway modes are as follows (in order of preference):
+
+1. Runway 35L/35R  
+2. Runway 17L/17R  
+3. Runways 13/31 (equal preference)
 
-When operationally required, runway 04 or 22 may be used, but consideration must be given to it's limited length.
+Crosswind operations are limited to **10KT**, and tailwind use is discouraged unless required.
+
+Runway 04/22 may only be used operationally and is **not available for circuit training**. Consideration must be given to it's limited length.
```

Comment:
I know that the metro D's have very specific local instructions but it might be beneficial to leave this as the standard crosswind limitation that all S1s and S2s learn from the Academy. Additionally, while circuit training isn't available IRL, given our traffic levels, it might be detrimental to deny an aircraft circuits despite being the only aircraft online, because the wind favours runway 04/22. Happy to discuss, but would probably lean towards allowing circuits to improve the pilot experience, and have ATC apply normal discretion in terms of workload, etc to determine whether circuits are available or not.